source/simulator/helper_functions.cc

```diff
@@ -1500,10 +1500,10 @@ namespace aspect
 
     // we use a different (potentially smaller) time step than in the advection scheme,
     // and we want all of our reaction time steps (within one advection step) to have the same size
-    const int number_of_reaction_steps = std::max((int) (time_step / parameters.reaction_time_step),
```

Comment:
@bangerth I do not think this code belongs here. Should I remove it from the my pull request?